apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -80,77 +82,93 @@ export function PoolListTable({
-              <span>Voting %</span>
-              {sortField == "votingShare"
-                ? OrderIcon(order)
-                : OrderIcon("neutral")}
-            </div>
-          </Table.HeaderCell>
-          <Table.HeaderCell onClick={() => handleSortingChange("apr")}>
-            <div className="flex gap-x-1 items-center">
-              <span> APR</span>
-              <Tooltip
-                content={`This is the APR calculate at the end of the round`}
-              >
-                <InfoCircledIcon />
-              </Tooltip>
-              {sortField == "apr" ? OrderIcon(order) : OrderIcon("neutral")}
-            </div>
-          </Table.HeaderCell>
-        </Table.HeaderRow>
-        <Table.Body>
-          {tableData.map((gauge) => (
-            <TableRow
-              key={gauge.poolId}
-              poolId={gauge.poolId}
-              network={gauge.network}
-              roundId={roundId}
-              symbol={gauge.symbol}
-              tvl={gauge.tvl}
-              votingShare={gauge.votingShare}
-              apr={gauge.apr}
-            />
-          ))}
-          <Table.BodyRow>
-            <Table.BodyCell colSpan={4}>
-              {isLoadingMore ? (
-                <Button
-                  className="w-full flex content-center justify-center gap-x-3 rounded-t-none rounded-b disabled:cursor-not-allowed"
-                  shade="medium"
-                  disabled={true}
+    <div className="flex flex-col justify-center text-white">
+      <div className="flex justify-center text-white shadow-lg mb-5">
+        <Table color="blue" shade={"darkWithBorder"}>
+          <Table.HeaderRow>
+            <Table.HeaderCell>Network</Table.HeaderCell>
+            <Table.HeaderCell>Composition</Table.HeaderCell>
+            <Table.HeaderCell classNames="text-end whitespace-nowrap">
+              Type
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              classNames="text-end whitespace-nowrap"
+              onClick={() => handleSortingChange("tvl")}
+            >
+              <div className="flex gap-x-1 items-center">
+                <Tooltip
+                  content={`This is the TVL calculate at the end of the round`}
                 >
-                  <Spinner size="sm" />
-                </Button>
-              ) : (
-                <Button
-                  className="w-full flex content-center justify-center gap-x-3 rounded-t-none rounded-b disabled:cursor-not-allowed"
-                  shade="medium"
-                  onClick={loadMorePools}
+                  <InfoCircledIcon />
+                </Tooltip>
+                <span>TVL</span>
+                {sortField == "tvl" ? OrderIcon(order) : OrderIcon("neutral")}
+              </div>
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              classNames="text-end whitespace-nowrap"
+              onClick={() => handleSortingChange("votingShare")}
```

Comment:
I think I said this, but this should refetch the page from the API instead of reordering things on the frontend -- you can reload the page with a query param with those filters and build the API request from that. That ensures the filters are also cached whenever uses them again.

Reply:
I'll do that on the filter task ([BAL-710](https://linear.app/bleu-llc/issue/BAL-710/ui-for-filter-on-pooltable))